pkg/utils/utils_test.go

```diff
@@ -0,0 +1,87 @@
+package utils
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSplitLines(t *testing.T) {
+	type scenario struct {
+		multilineString string
+		expected        []string
+	}
+
+	scenarios := []scenario{
+		{
+			"",
+			[]string{},
+		},
+		{
+			"\n",
+			[]string{},
+		},
+		{
+			"hello world !\nhello universe !\n",
+			[]string{
+				"hello world !",
+				"hello universe !",
+			},
+		},
+	}
+
+	for _, s := range scenarios {
+		assert.EqualValues(t, s.expected, SplitLines(s.multilineString))
+	}
+}
+
+func TestWithPadding(t *testing.T) {
+	type scenario struct {
+		str      string
+		padding  int
+		expected string
+	}
+
+	scenarios := []scenario{
+		{
+			"hello world !",
+			1,
+			"hello world !",
+		},
+		{
+			"hello world !",
+			14,
+			"hello world ! ",
+		},
+	}
+
+	for _, s := range scenarios {
+		assert.EqualValues(t, s.expected, WithPadding(s.str, s.padding))
+	}
+}
+
+func TestCurrentRepoName(t *testing.T) {
```

Comment:
CurrentRepoName() expects GitCommand.navigateToRepoRootDirectory() to have been called (it's called at program startup) so in this case it actually gives an incorrect result: we would expect it to return 'lazygit' rather than 'utils'. I'm thinking we should just rename the function to 'CurrentDirectoryName' to make it more accurately reflect what it's doing

Reply:
From this location it returns that result so I only can remove this test.